packages/gatsby/src/redux/run-sift.js

```diff
@@ -125,17 +129,14 @@ const filterWithoutSift = (filters, nodeTypeNames, filtersCache) => {
   return result
 }
 
-// Not a public API
-exports.filterWithoutSift = filterWithoutSift
```

Comment:
lol?

Reply:
Moved to the bottom :) It's used for tests